code-studio/src/js/header.js

```diff
@@ -129,31 +129,12 @@ header.build = function (stageData, progressData, currentLevelId, scriptName, pu
   }
 
   var popupLoaded = false;
-  function lazyLoadPopup(trophiesClicked) {
+  function lazyLoadPopup() {
     if (!popupLoaded) {
       popupLoaded = true;
-      $.ajax({
-        url: "/popup/stats",
-        data: {
-          script_id: stageData.script_id,
-          current_level_id: currentLevelId,
-          user_id: clientState.queryParams('user_id'),
```

Comment:
Looks like losing this parameter regressed the dropdown when a teacher is viewing a student's results.  Working on a fix separately.